### DISCUSSION TITLE
PIN layout and device profile

### DISCUSSION THREAD

**Discussion by basicnulldev** [tbnobody/OpenDTU#1058] [Q&A] [answered]
Folks, I am new to this, please bare with me if I am doing stuff wrongly...

Managed to get openDTU on my ESP-WROOM32 via VScode, yay! But I am now strugling with wiring of the pins to get my display and antenna working.

Thing is, my board has only 30 pins. I haven't found any documentation of the board. On it's back it says:

> HW-463C V0.0.6
> NodeMCU-ESP32
> ESP32 DEVKITV!

PIN's are named D12, D13 etc. Any hint on where I find a documentation of that board is welcomed as well as a step by step guide to get that device priofile onto the device.

Thanks in advance!

**Comment by CBFOUR** [comment, marked as answer]
https://www.google.com/search?q=esp32+devkitc+pinout

![esp32_pinout](https://github.com/tbnobody/OpenDTU/assets/128149877/ebfd568d-6ca2-44ca-9c4c-6a3612c1ba16)



![ESP32_Wiring](https://github.com/tbnobody/OpenDTU/assets/128149877/74e01211-95d8-401d-9954-73f74a046d56)
Look for the GPIO , they should match


Cheers

**Reply by basicnulldev** [reply]
That was super helpful and played the trick! Many thanks!
